fn main() -> Result<(), u8> {
   return Err(1);
}
